src/test/resources/annotation-job.properties

```diff
@@ -31,4 +31,4 @@ config.db.read-preference=primary
 
 db.collections.features.name=features
 db.collections.stats.name=populationStatistics
-db.name=AnnotationJobTest
+spring.data.mongodb.database=AnnotationJobTest
```

Comment:
Yes correct. With the annotation loader step migrated (@jmmut) then this can be removed